### PR TITLE
Download the RDL binary from Github

### PR DESCRIPTION
**Pull request by yamlin**

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: parsec-gradle-plugin/build.gradle [hunk 1]
```diff
@@ -72,8 +66,29 @@ task downloadParsecRdl(type: Exec) {
     }
 }
 
-jar.dependsOn downloadParsecRdl
-publishPlugins.dependsOn downloadParsecRdl
-publishPlugins.mustRunAfter check
-postProcess.dependsOn publishPlugins
-test.dependsOn downloadParsecRdl
+task downloadRdlBinary(type: Exec) {
+    def osName = System.getProperty("os.name").equals("Mac OS X") ? "darwin" : "linux";
```
Comment (shichung) [inline parsec-gradle-plugin/build.gradle:70]: does "equals("Mac OS X")" here mean this check is limited to the specific version of Mac OS ?
would it be better to just check "SystemUtils.IS_OS_MAC"?

reference:
http://stackoverflow.com/questions/4689240/detecting-the-platform-window-or-linux-by-groovy-grails

Comment (chinkong83) [reply parsec-gradle-plugin/build.gradle:70]: SystemUtils 是 apache 出的 util，not standard

Comment (shichung) [reply parsec-gradle-plugin/build.gradle:70]: yes, I know it, thanks for the reminder

**Comment by shichung** [comment]
LGTM

File: parsec-gradle-plugin/build.gradle (hunk 1 above)
Comment (shichung) [reply parsec-gradle-plugin/build.gradle:70]: the team members discussed it and decided not to fix it in this phase

**Pull request merged**